trade-finance-manager-ui/templates/utilisation-reports/record-corrections/check-the-information.njk

```diff
@@ -92,26 +92,27 @@
     }
   }) }}
 
-  <form method="post">
-    <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
-    <div class="govuk-button-group">
+  <div class="govuk-button-group">
+    <form method="post">
+      <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
       {{ govukButton({
         text: "Send record correction request",
         attributes: {
           'data-cy': 'continue-button'
         }
       }) }}
-
-      {{ govukButton({
-        text: "Cancel record correction request",
-        href: cancelLink,
-        classes: "govuk-button--secondary",
-        attributes: {
-          'data-cy': 'cancel-button'
-        }
-      }) }}
-    </div>
-  </form>
+    </form>
+    <form method="post">
+      <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+      <input
+        class="govuk-button govuk-button--secondary"
```

Comment:
can this be a govukButton?

Reply:
Tried this out initially, but as we're posting to this url we need the `formaction` attribute (versus using a `href`) which isn't supported by govukButton

This seems to be the convention in other TFM UI and portal UI templates when we need to post, so went with this approach